integration_tests/__tests__/__snapshots__/transform-test.js.snap

```diff
@@ -0,0 +1,27 @@
+exports[`babel-jest instruments only specific files and collects coverage 1`] = `
```

Comment:
is that an expression of excitement about snapshot testing or did i mess something up? :D